{{cookiecutter.repo_name}}/frontend/js/main.ts

```diff
@@ -7,20 +8,39 @@ import "./components/modal";
 import "./components/flyout";
 import "./links";
 
-// @ts-expect-error // needs to declare that htmx lives on window, auto added by import
-const { htmx } = window; // eslint-disable-line  @typescript-eslint/no-unused-vars
-
 // @ts-expect-error // this whole system is broken w/ vite
 if (import.meta.env.MODE !== "development") {
   // // @ts-expect-error  // this whole system is broken w/ vite
   // import("vite/modulepreload-polyfill"); // eslint-disable-line import/no-unresolved
   // https://github.com/vitejs/vite/issues/4786
 }
 
-// TODO: when  https://github.com/vitejs/vite/issues/5808 is fixed, setup HMR for html through htmx.ajax
```

Comment:
What happened here on the issue? I don't see any additional fix after the patch you said still had it broken?